Makefile

```diff
@@ -29,20 +30,26 @@ else ifeq (MINGW,$(findstring MINGW,$(detected_OS)))
 	# MINGW detected => NO SUPPORT FOR BUILDING CERES SOLVER  (yet)
 	CERES := OFF
 	OMFIT := OFF
+    ABI := WINDOWS64
 else
 	BUILD_DIR := build/linux
 	INSTALL_DIR := install/linux
+    ABI := LINUX64
```

Comment:
This doesn't work:
```bash
OMSimulator/OMTLMSimulator/3rdParty/../Makefile.head:7: *** Please set the ABI environment variable, typical choices are: WINDOWS32, WINDOWS64, or LINUX64.  Stop.
```

Reply:
I replaced the four spaces with a tab. It seems to work when I try it. I first did not notice it, since I had ABI defined as an environment variable.

Reply:
No, it doesn't work either.

Reply:
That's weird. If I comment that line out, I get the same error as you. If not, compilation works just fine. Are you compiling from top level directory, and using "make OMTLMSimulator"?

Reply:
Yes.